pkg/web_css/lib/src/_base.scss

```diff
@@ -485,3 +490,39 @@ pre > code {
 .center {
   text-align: center;
 }
+
+.pub-modal {
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  top: 0;
+  background: rgba(0, 0, 0, 0.7);
```

Comment:
Lovely! :)

nit: what about `background: white`, and then `opacity: 0.7` and `transition: opacity 80ms`, just to give a smooth feel :)

Reply:
Maybe... but if we also want to use this or a similar thing for the "please wait" indication, then it may not be good to have the transition there. Let's postpone this a bit.